src/utils/sanitizeFilename.js

```diff
@@ -11,7 +10,7 @@ export default function(platform, str) {
 
   // spaces will cause problems with Ubuntu when pinned to the dock
   if (platform === 'linux') {
-    return _.kebabCase(result);
+    return result.replace(' ', '');
```

Comment:
This will only replace the first occurence of a space, not all.

Reply:
@ronjouch Please check the regex in the new commit to perform the same.